pkgs/by-name/im/immich-public-proxy/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nixosTests,
+  nodejs,
+}:
+let
+  pname = "immich-public-proxy";
+  version = "1.5.3+unstable=2024-12-02";
+  repo = fetchFromGitHub {
+    owner = "alangrainger";
+    repo = pname;
```

```suggestion
    repo = "immich-public-proxy";
```
see https://github.com/NixOS/nixpkgs/issues/277994